api/proto/message/message.proto

```diff
@@ -45,25 +36,24 @@ enum MessageType {
 //
 // The request field will be either one of the structure corresponding to the MessageType type.
 message Message {
-  ReceiverType receiver_type = 1;
```

Comment:
why removing ReceiverType? I think it's still beneficial

Reply:
it's not used anywhere since we can determine the receiver by their bls public key. we always need to check the key validity instead of the message type because the message type can be construct by the validator arbitrarily